LocalPackages/SubscriptionUI/Package.swift

```diff
@@ -12,13 +12,14 @@ let package = Package(
             targets: ["SubscriptionUI"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/duckduckgo/BrowserServicesKit", exact: "185.0.0"),
+        .package(url: "https://github.com/duckduckgo/BrowserServicesKit", revision: "0e9f45c1612c27aba128ba2142caf984362c95db"),
```

Comment:
One note to hopefully save you some time, you won't need to do this if you instead specify BSK as a branch in the Xcode project settings. I do that a lot and it works great, Xcode seems smart enough to supersede the values like this one, believe it or not.